paasta_tools/utils.py

```diff
@@ -707,6 +707,12 @@ def default(text):
             'help': 'Logs from security-related services such as firewall monitoring',
         },
     ),
+    (
+        'oom', {
```

Comment:
Can you add this to the list of DEFAULT_COMPONENTS so people will see them without any extra paasta logs cli args?